### PR TITLE
US National Archives Research Catalog.js -- fix eslint warnings

### PR DESCRIPTION
**Pull request by retorquere**

### CODE REVIEW

**Comment by retorquere** [comment]
updated for @zotero/eslint-config@1.0.2

File: US National Archives Research Catalog.js [hunk 1]
```diff
@@ -162,5 +164,5 @@ var testCases = [
 			}
 		]
 	}
-]
+];
```
Comment (zuphilip) [inline US National Archives Research Catalog.js]: no

Comment (retorquere) [reply US National Archives Research Catalog.js]: I'm truly sorry about these -- I thought I had grepped out all instances. Would it be possible to merge https://github.com/zotero/translators/pull/1958? That would immediately flag them.

Comment (zuphilip) [reply US National Archives Research Catalog.js]: Don't be sorry. It is easy to spot for me and I guess easy to fix for you.

**Comment by zuphilip** [comment]
Thank you!

**Pull request merged**